time_tracker_api/time_entries/time_entries_model.py

```diff
@@ -282,19 +300,18 @@ def checks_owner_and_is_not_started(cls, data: dict):
                 "The specified time entry is already running",
             )
 
-    def get_all(self, conditions: dict = {}) -> list:
+    def get_all(self, conditions: dict = None, **kwargs) -> list:
         event_ctx = self.create_event_context("read-many")
         conditions.update({"owner_id": event_ctx.user_id})
 
-        date_range = self.handle_date_filter_args(args=conditions)
-        time_entries = self.repository.find_all(
-            event_ctx, conditions=conditions, date_range=date_range
-        )
-
-        project_dao = projects_model.create_dao()
-        projects = project_dao.get_all()
-        add_project_name_to_time_entries(time_entries, projects)
-        return time_entries
+        if "start_date" and "end_date" in conditions:
```

Comment:
I think it would better If we move the logic inside this if to the function `handle_date_filter_args`

Reply:
ok, cool